src/Pact/Types/Command.hs

```diff
@@ -149,8 +144,8 @@ verifyCommand orig@Command{..} = case (ppcmdPayload', ppcmdHash', mSigIssue) of
     ppcmdPayload' = traverse parsePact =<< A.eitherDecodeStrict' _cmdPayload
     parsePact :: Text -> Either String ParsedCode
     parsePact code = ParsedCode code <$> parseExprs code
-    (ppcmdSigs' :: [(UserSig,Bool)]) = (\u -> (u,verifyUserSig _cmdHash u)) <$> _cmdSigs
-    ppcmdHash' = verifyHashTx _cmdHash _cmdPayload requestKeyHash
+    (ppcmdSigs' :: [(UserSig,Bool)]) = (\u -> (u,verifyUserSig (toUntypedHash _cmdHash) u)) <$> _cmdSigs
```

Comment:
See comment regarding conversion between PactHash and Hash.

Reply:
So, there is a good reason to still have `Scheme` implementations validate an untyped hash, as the hash value is signable no matter what created it, and it's even possible that someday we might sign something other than a Command hash. Meanwhile everything in Command and Pact is typed to PactHash.

Regarding keeping SHA3_256, that's future-proofing us when we want to offer other hashing algorithms to users within pact.